xpublish/rest.py

```diff
@@ -28,22 +29,32 @@ def __init__(self, xarray_obj):
 
         self._app = None
         self._app_kws = {}
-        self._app_routers = [common_router, base_router, zarr_router]
+        self._app_routers = [
+            (common_router, {}),
+            (base_router, {'tags': ['info']}),
+            (zarr_router, {'tags': ['zarr']}),
+        ]
 
         self._cache = None
         self._cache_kws = {'available_bytes': 1e6}
 
         self._initialized = False
 
-    def __call__(self, cache_kws=None, app_kws=None):
+    def __call__(self, routers=None, cache_kws=None, app_kws=None):
         """
         Initialize this RestAccessor by setting optional configuration values
 
         Parameters
         ----------
-        cache_kws : dict
+        routers : list, optional
+            A list of :class:`fastapi.APIRouter` instances to include in the
+            fastAPI application. If None, the default routers will be included.
+            The items of the list may also be tuples with the following format:
+            ``[(router1, {'prefix': '/foo', 'tags': ['foo', 'bar']})]``.
+            This is useful for passing arguments to :meth:`fastapi.FastAPI.include_router`.
```

Comment:
It would be good to clarify a bit here that the form of the tuples is `(APIRouter, Dict)` where the contents of the `Dict` are passed to `includer_router` as `kwargs`.